build.gradle

```diff
@@ -186,7 +186,6 @@ dockerCompose {
       "l1-node-genesis-generator",
       "l1-el-node",
       "l1-cl-node",
-      "l2-node",
```

Comment:
Curious, do we also need `shomei-sr` and` zkbesu-shomei-sr` here?

Reply:
We don't. We use fakes. 
There is a test that uses realone but call dokcer compose to restart those ones on demand.